### PR TITLE
Upgrade to ocamlformat 0.13.0

### PR DESCRIPTION
**Pull request by gpetiot**
No option was added or removed between 0.12 and 0.13, there were only consistency fixes, so the upgrade should be painless.

### CODE REVIEW

File: src/dune/coq_lib_name.ml [hunk 1]
```diff
@@ -1,29 +1,32 @@
 (* This file is licensed under The MIT License *)
 (* (c) MINES ParisTech 2018-2019               *)
 (* (c) INRIA 2020                              *)
-(*     Written by: Emilio Jesús Gallego Arias  *)
+(* Written by: Emilio Jesús Gallego Arias *)
```
Comment (ejgallego) [inline src/dune/coq_lib_name.ml]: This seems suspicious

File: src/dune/coq_lib_name.ml [hunk 2]
```diff
@@ -1,29 +1,32 @@
...
 let wrapper x = to_string x
 
 let decode : (Loc.t * t) Dune_lang.Decoder.t =
-  Dune_lang.Decoder.plain_string (fun ~loc s ->
-    (loc, String.split ~on:'.' s))
+  Dune_lang.Decoder.plain_string (fun ~loc s -> (loc, String.split ~on:'.' s))
 
 let encode : t Dune_lang.Encoder.t =
-  fun lib -> Dune_lang.Encoder.string (to_string lib)
+ fun lib -> Dune_lang.Encoder.string (to_string lib)
```
Comment (ejgallego) [inline src/dune/coq_lib_name.ml]: Is this intended?

File: src/dune/coq_rules.ml [hunk 3]
```diff
@@ -109,7 +115,12 @@ let setup_rule ~expander ~dir ~cc ~source_rule ~coq_flags ~file_flags
   let dir = Path.build dir in
   let file_flags = file_flags @ [ Command.Args.Dep (Path.build source) ] in
   let cd_arg = Command.Args.As [ "-dyndep"; "opt" ] :: file_flags in
-  let cd_arg = if boot then Command.Args.As ["-boot"] :: cd_arg else cd_arg in
+  let cd_arg =
```
Comment (ejgallego) [inline src/dune/coq_rules.ml]: I count 78 chars here, is this break intended?

Comment (Julow) [reply src/dune/coq_rules.ml]: It seems that line was not formatted before. Adding spaces inside the `[ .. ]` forces the break.

Comment (ejgallego) [reply src/dune/coq_rules.ml]: I see, thanks!

File: src/dune/coq_lib_name.ml (hunk 1 above)
Comment (Julow) [reply src/dune/coq_lib_name.ml]: I get the same diff with OCamlformat 0.12.
This is caused by the `wrap-comments=true` option in `.ocamlformat`. Removing it doesn't add more diffs.

Comment (ejgallego) [reply src/dune/coq_lib_name.ml]: Ok thanks, I guess I'll adapt the end-of-comment wrapping then.

File: src/dune/coq_lib_name.ml (hunk 2 above)
Comment (Julow) [reply src/dune/coq_lib_name.ml]: This is. This would format the same with 0.12 or 0.13.

**Comment by diml** [comment]
@gpetiot could you sign your commit?

**Comment by diml** [comment]
Thanks!

**Pull request merged**